src/main/java/com/nike/cerberus/command/core/GenerateCertsCommand.java

```diff
@@ -0,0 +1,177 @@
+                    "subject name {env}.{base-domain} and with the following subject alternative name {env}.{region}.{base-domain}\n" +
+                    "ex: cerberus -e demo -r us-west-2 generate-certs -d cerberus-oss.io would make a cert for demo.cerberus-oss.io " +
+                    "with a sans of demo.us-west-2.cerberus-oss.io so that we can create a CNAME record for " +
+                    "demo.cerberus-oss.io that will point to an ALB in us-west-2 with a CNAME record of " +
+                    "demo.us-west-2.cerberus-oss.io and the cert will be valid for both",
+            required = true
+    )
+    private String baseDomainName;
+
+    public String getBaseDomainName() {
+        return baseDomainName;
+    }
+
+    @Parameter(
+            names = {
+                    HOSTED_ZONE_ID_LONG_ARG,
+                    HOSTED_ZONE_ID_SHORT_ARG
+            },
+            description = "The AWS Route 53 hosted zone id that is configured to create records for the base domain",
+            required = true
+    )
+    private String hostedZoneId;
+
+    public String getHostedZoneId() {
+        return hostedZoneId;
+    }
+
+    @Parameter(
+            names = {
+                    ADDITIONAL_SUBJECT_ALT_NAME_LONG_ARG,
+                    ADDITIONAL_SUBJECT_ALT_NAME_SHORT_ARG
+            },
+            description = "Alternative subject names, this should be any additional cnames that need to be secured. such as "
+    )
+    private List<String> subjectAlternativeNames = new ArrayList<>();
+
+    public List<String> getSubjectAlternativeNames() {
+        return subjectAlternativeNames;
+    }
+
+    @Parameter(
+            names = {
+                    ENABLE_LE_CERTFIX_LONG_ARG,
+                    ENABLE_LE_CERTFIX_SHORT_ARG
+            },
+            description = "This command uses the acme4j client to communicate with the ACME server, " +
+                    "it supports uses a hardcoded letsencrypt cert to get around ssl errors, you can use this " +
+                    "flag if your truststore is not configured to trust LE Certs, which can be found " +
+                    " here: https://letsencrypt.org/certificates/"
+    )
+    private boolean EnableLetsEncryptCertfix = false;
+
+    public boolean enableLetsEncryptCertfix() {
+        return EnableLetsEncryptCertfix;
+    }
+
+    @Parameter(
+            names = {
+                    CERT_FOLDER_LONG_ARG,
+                    CERT_FOLDER_SHORT_ARG
+            },
+            description = "The cert folder to store the generated files",
+            required = true
+    )
+    private String certDir;
+
+    public String getCertDir() {
+        return certDir;
+    }
+
+    @Parameter(
+            names = {
+                    ACME_API_LONG_ARG,
+                    ACME_API_SHORT_ARG
+            },
+            description = "The ACME API Url to use, This can be any ACME provider like LetsEncrypt: "  + LETS_ENCRYPT_ACME_API_URI,
```

Comment:
Change to "The ACME provider API URL to use, e.g. Let's Encrypt:"

Reply:
- [ ] Done